diff --git a/apps/next/src/app/playground/_components/forms/RealtimeClientForm.tsx b/apps/next/src/app/playground/_components/forms/RealtimeClientForm.tsx
index 8234950..02b330f 100644
--- a/apps/next/src/app/playground/_components/forms/RealtimeClientForm.tsx
+++ b/apps/next/src/app/playground/_components/forms/RealtimeClientForm.tsx
@@ -25,6 +25,7 @@ import {
   SelectTrigger,
   SelectValue,
 } from '@/components/ui/select'
+import { PUBLIC_REALTIME_URL } from '@/lib/constants'
 import { transformOptionalNumber } from './helpers'
 
 type Props = {
@@ -37,7 +38,8 @@ export function RealtimeClientForm({ disabled, status, logger }: Props) {
   const { supabaseUrl, supabaseKey } = useEnv()
 
   const onSubmit = (options: RealtimeClientFormValues) => {
-    useRealtimeStore.getState().create(`${supabaseUrl}/realtime/v1`, {
+    const endpoint = PUBLIC_REALTIME_URL || `${supabaseUrl}/realtime/v1`
+    useRealtimeStore.getState().create(endpoint, {
       ...realtimeOptions(options, supabaseKey),
       logger,
     })
